modules/creating_a_module.md

```diff
@@ -35,20 +35,28 @@ A JHipster module:
 - follows an extension of the Yeoman rules listed at [http://yeoman.io/generators/](http://yeoman.io/generators/) and can be installed, used and updated using the "yo" command. Instead of being prefixed by "generator-", it is prefixed by "generator-jhipster-", and instead of having just the "yeoman-generator" keyword, it must have 2 keywords, "yeoman-generator" and "jhipster-module".
 - A JHipster module registering as a hook should not call `process.exit` in its generators being hooked.
 
-## Composability
+## Import the generator-jhipster
 
-A JHipster module uses the new "composability" feature from Yeoman, described at [http://yeoman.io/authoring/composability.html](http://yeoman.io/authoring/composability.html) to have access to JHipster's variables and functions.
+A JHipster module must import the generator-jhipster:
 
-For this, it composes with the "jhipster:modules" sub generator:
+```
+    const util = require('util');
+    const BaseGenerator = require('generator-jhipster/generators/generator-base');
+    const jhipsterConstants = require('generator-jhipster/generators/generator-constants');
+
+    const JhipsterGenerator = generator.extend({});
+    util.inherits(JhipsterGenerator, BaseGenerator);
+
+    module.exports = JhipsterGenerator.extend({
+
+        // all your yeoman code here
+
+    });
+```
 
-    compose: function() {
-        this.composeWith('jhipster:modules', {
-            options: {
-                jhipsterVar: jhipsterVar,
-                jhipsterFunc: jhipsterFunc
-            }
-        });
-    },
+Then, you can access to JHipster's variables and functions directly.
+
+**Note**: you don't need to use the composability described at [http://yeoman.io/authoring/composability.html](http://yeoman.io/authoring/composability.html) any more.
```

Comment:
I don't think we have to tell this, people can still use composability for other purposes.